src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss

```diff
@@ -12,7 +12,269 @@
     padding-top: 200px;
 }
 
+.img404 {
+    margin-bottom: 35px;
+
+    position: relative;
+    z-index: 1;
+}
+
+.text404 {
+    position: relative;
+    z-index: 99;
+}
+
 .honeyLost {
     font-family: "geometriaBold";
     font-size: 4rem;
+    line-height: 1em;
+
+    position: relative;
+    z-index: 1;
+}
+
+.tears {
+    position: absolute;
+    left: 50%;
+    top: 50px;
+    z-index: 2;
+}
+
+.tearDrop {
+    @mixin animation_name(tears);
+    @mixin animation_duration(4s);
+    @mixin animation_timing_function(linear);
+}
+
+@-webkit-keyframes tears {
+    0% {
+        -webkit-transform: scale(.25) translate(0, 0);
+    }
+    100% {
+        -webkit-transform: scale(.75) translate(0, 1000px);
+    }
+}
+
+@keyframes tears {
+    0% {
+        -moz-transform: scale(.25) translate(0, 0);
```

Comment:
I will try to do something for that, but I think there's a reason why we can. Let's see